packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.tsx

```diff
@@ -209,7 +215,7 @@ export class ContextualMenu extends BaseComponent<IContextualMenuProps, IContext
           onDismiss={ this.props.onDismiss }
           bounds={ bounds }
           directionalHintFixed={ directionalHintFixed }>
-          <div ref={ (host: HTMLDivElement) => this._host = host } id={ id } className={ css('ms-ContextualMenu-container', className) }>
+          <div style={ { width: contextMenuWidth || 'auto'} } ref={ (host: HTMLDivElement) => this._host = host } id={ id } className={ css('ms-ContextualMenu-container', className) }>
```

Comment:
Doing this now

    let contextMenuStyle;
    if (useTargetWidth && this._target as HTMLElement) {

        let contextMenuWidth = (this._target as HTMLElement).offsetWidth;
        contextMenuStyle = {
            width: contextMenuWidth
        };
    }